ProductionSystem/DB/ProductionDB.py

```diff
@@ -166,6 +188,60 @@ def getProductionParameters(self, prodName, parameters, connection=False):
       return S_OK(paramDict[reqParam])
     return S_OK(paramDict)
 
+  ###########################################################################
+  #
+  # These methods manipulate the ProductionSteps table
+  #
+
+  def getProductionStep(self, stepID, connection=False):
+    """ It returns the ProductionStep corresponding to the stepID
+
+    :param stepID: the ID of the production Step stored in the ProductionSteps table
+    :return: the attributes of Production Step corresponding to the stepID
+    """
+    connection = self.__getConnection(connection)
+    req = "SELECT %s FROM ProductionSteps WHERE StepID = %s" % (intListToString(self.PRODSTEPSPARAMS), str(stepID))
+    res = self._query(req, connection)
+    if not res['OK']:
+      return res
+    if not res['Value']:
+      return S_ERROR("ProductionStep %s did not exist" % str(stepID))
+    return S_OK(res['Value'][0])
+
+  def addProductionStep(self, stepName, stepDescription, stepLongDescription, stepBody, stepType, stepPlugin,
+                        stepAgentType, stepGroupSize, stepInputquery, stepOutputquery, connection=False):
+    """ Add a Production Step
+
+    :param stepName:
+    :param stepDescription:
+    :param stepLongDescription:
+    :param stepBody:
+    :param stepType:
+    :param stepPlugin:
+    :param stepAgentType:
+    :param stepGroupSize:
+    :param stepInputquery:
+    :param stepOutputquery:
+    :return:
```

Comment:
Also these ones should be filled. BTW please include also the type, e.g.
```
:param str stepName:
```
here, and in other docstrings.